flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "only-arches": ["x86_64"],
```

Comment:
why not aarch64?

Reply:
so building from source still not tried? that's very disappointing.

Reply:
I'd say I personally don't have this spare time, so apart from any upstream effort I can only do what I can do.